packages/core/src/types.ts

```diff
@@ -48,7 +48,7 @@ export interface IntrinsicSxElements {
   em: JSX.IntrinsicElements['em'] & SxProps
   strong: JSX.IntrinsicElements['strong'] & SxProps
   div: JSX.IntrinsicElements['div'] & SxProps
-  delete: JSX.IntrinsicElements['div'] & SxProps
```

Comment:
I think this is a MDX thing as is `inlineCode` and `thematicBreak`.
https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/mdx/customizing-components.md

Reply:
I guess I'm just a bit confused- in v0.3 (which fixed the issue reported in #401), `delete` was renamed to `del`.

Reply:
I didn’t know about #401. I’m as confused as you are or more. Sorry 😄